src/Akka.Persistence.Sql/Journal/SqlWriteJournal.cs

```diff
@@ -54,6 +54,7 @@ public class SqlWriteJournal : AsyncWriteJournal
         public SqlWriteJournal(Configuration.Config journalConfig)
         {
             _log = Context.GetLogger();
+            var selfUuid = Guid.NewGuid().ToString("N");
```

Comment:
Actual implementation, writer uuid is generated every time the write journal is instantiated.